lib/node_modules/@stdlib/blas/ext/base/dnanasum/benchmark/benchmark.js

```diff
@@ -39,18 +40,15 @@ var dnanasum = require( './../lib/dnanasum.js' );
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x;
-	var i;
+	var x = filledarrayBy( len, 'float64', clbk );
+	return benchmark;
```

Comment:
why are you returning benchmark here?

Reply:
It seems to confuse me. Is it not returning benchmark here. To update this package, I have to use a utility function for generating random numbers and remove the loop. I've done that, and the benchmark is returning as it was before. I only made changes to the for loop part, which was  
```
x = new Float64Array( len );
for ( i = 0; i < x.length; i++ ) {
	if ( randu() < 0.2 ) {
		x[ i ] = NaN;
	} else {
		x[ i ] = ( randu()*20.0 ) - 10.0;
}
```